app/models.py

```diff
@@ -648,6 +648,7 @@ class ResponseTokens(db.Model):
     def __init__(self,
                  response_id,
                  expiration_date=None):
+        from app.lib.date_utils import calendar
```

Comment:
Does this import still need to go here instead of back on top since you moved NYCHolidays out of date_utils?